setupTests-integration.js

```diff
@@ -15,16 +17,24 @@ function getTestNameFromGlobalJasmineInstance() {
 
 // Mock config file
 jest.mock(
-  "./src/config",
-  () => require("./__tests__/integration/configuration/config-fixture.js"),
-  { virtual: true }
+  './src/config',
+  () => {
+    let config = require('./__tests__/integration/configuration/config-fixture');
```

Comment:
After the application refactor, all tests ( integration and mocked ) stopped working because of this mock file code on the setup. Sometimes the retrieved config file was correct, but in other cases it was nested inside a `default` property on the required js.

I believe it is related to the config and constants being referred not only by the `index.js` file anymore, but from `js` files spread on various folders with different relative addresses. The simplest solution I found was to make this check for the `default` property on the required files.

Since this is directly related to the refactor of the application, I believe it belongs to this PR. Do you agree?

Reply:
Yes, I agree.